app-misc/tinysparql/tinysparql-3.10.1.ebuild

```
@@ -38,19 +38,18 @@ BDEPEND="
 	$(vala_depend)
```

```suggestion
	vala? ( $(vala_depend) )
```

Vala is conditional and builds fine without it

---

Don't know how to add the suggestions for the parts that are not shown in github's diff page but do the same in `src_prepare()` `use vala && vala_setup`